dynamicdns.bash

```diff
@@ -258,6 +266,33 @@ function submitApiRequest {
   printf "$RESPONSE" | grep "^success$" > /dev/null
 }
 
+function listRecord {
```

Comment:
This is mostly a copy of the `deleteRecord` function. I think it would be a better choice to modify that function to return early if we're in this mode rather than continuing to delete the record so that we can avoid making an additional API call unnecessarily.

Alternatively, we could refactor to make the delete method take a parameter that is the value of the current record so that listing and deleting are still two separate functions (which is better for single responsibility), but still avoid the duplicate API calls.

Reply:
Yeah, that makes sense. I will have a look at this on Tuesday.